tests/subsys/net/lib/lwm2m/lwm2m_registry/src/lwm2m_registry.c

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/ztest.h>
+#include <zephyr/fff.h>
```

Comment:
I don't see any stubs, is the `fff.h` unneeded here?

Reply:
Oh, that's true.